### PR TITLE
Role-Assignment

### PR DESCRIPTION
**Pull request by Snehal-Reddy**
few dummy functions created and role_selector for home free kick done .

### CODE REVIEW

File: gameAnalyzer.py [hunk 1]
```diff
@@ -12,37 +16,59 @@ def __init__(self, tasklist, state, play=None):
...
                 tasks = self.intersection(requiredTasks, self.tasklist)
                 # task is dict, keyword -> tasks name, value -> their target point
                 tasks = self.targetPoints(tasks)
                 bots = self.availableBots()
                 return tasks, bots
+
         else:
             print("this static play is not available!!")
 
+def main():
```
Comment (sipah00) [inline gameAnalyzer.py]: main function would be in `test.py`, here we will write `class` and `member` functions, in `test.py` we will `import` this file and then in callback of `belief_state`, make instance of `gameAnalyzer` and will call to `staticPlays`.

File: tasks/attacker.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+from skills import sKickToPoint
+
+
+
+
+class TAttacker(object):
+	def __init__(self, arg):
+		super(TAttacker, self).__init__()
+		self.arg = arg
+	def execute(self,state,bot_id,play_type=0):
```
Comment (sipah00) [inline tasks/attacker.py]: as we have discussed, we are not going to write any execute function here, only write `logic` for target point in `class function`, (like, in TAttacker, you write logic for target point and then you call `sGoToBall.execute()` here we have to write only that logic and return target point as function output).

**Review by sipah00 — DISMISSED** [review]
I have added some imp points, do the required changes, then it is good to go.

**Comment by sipah00** [comment]
@Snehal-Reddy @RKJenamani @VISHAL-RAJ-RAJ looks good, are all tasks working? add some test cases to make sure that every thing works fine, are tasks written with fuzzy logic working properly?

**Comment by Snehal-Reddy** [comment]
They are yet to be tested . We will do it tomorrow in the lab

File: tasks/attacker.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+#VERY IMPORTANT -
+# take into consideration that there can be more than 2 attackers
+
+class TAttacker(object):
+	def __init__(self):
+		super(TAttacker, self).__init__()
+	def getTargetPos(self,state,play=-1,number):
+		ballPos = Vector2D(int(state.ballPos.x), int(state.ballPos.y))
+		x = ballPos.x
```
Comment (sipah00) [inline tasks/attacker.py:18]: lines `18, 19 and 20` are unnecessary.

File: tasks/distractor.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+#VERY IMPORTANT -
+# take into consideration that there can be more than 2 Distractors
+
+class TDistractor(object):
+	def __init__(self, arg):
+		super(TDistractor, self).__init__()
+		self.arg = arg
+	def getTargetPos(self,state,play=-1,number):
+		x = 0
```
Comment (sipah00) [inline tasks/distractor.py:18]: why is it hard-coded? add the logic for `Distracter` here, discuss among teammates for logic and ideas.

File: tasks/marker_fuzz.py [hunk 5]
```diff
@@ -0,0 +1,345 @@
...
+	print "#"*50
+	print T
+	print "BOT_TO_PASS---->"
+	return index, max_score
+
+
+
+
+
+
```
Comment (sipah00) [inline tasks/marker_fuzz.py:315]: remove these lines

**Comment by Snehal-Reddy** [comment]
Yeah . It was discussed in the meeting yesterday . We explained the current code and distributed tasks to all the people . So by the end of summer , both execute and getTargetPos functions will be ready . We only put some vague values .

**Comment by sipah00** [comment]
#3

**Pull request closed without merging**